README.md

```diff
@@ -10,7 +10,7 @@ Finished certifications is checked:
 
 ## Reactjs.org
 
-Tic-tac-toe tutorial game is built and its source code [here](TODO:Add relative path link from the repository). Implementation goals:
+Tic-tac-toe tutorial game is built and its source code [/my-app/](TODO:Add relative path link from the repository). Implementation goals:
```

Comment:
it should be like that: `[here](/my-app/)`. look at how to add link in markdown: https://www.markdownguide.org/basic-syntax/#links